udp_mux.go

```diff
@@ -23,6 +23,7 @@ type UDPMux interface {
 	GetConn(ufrag string, addr net.Addr) (net.PacketConn, error)
 	RemoveConnByUfrag(ufrag string)
 	GetListenAddresses() []net.Addr
+	ConnCount() int
```

Comment:
Unfortunately, this breaks the API :(

We need to bump the major version then.

Reply:
That's a question... though I think there is no developer has implemented his own UDPMux but keep API compatibility will be good. I will try to import a new interface to the port balance feature and only enable it if the child Muxes support the new API to keep the `UDPMux` unchanged.

Reply:
good catch @stv0g! It'd be good to keep UDPMux interface simpler without this additional requirement.